tests/e2e/no_hci_mesh_rwx_singlevc_topology_disruptive.go

```diff
@@ -0,0 +1,726 @@
+	admissionapi "k8s.io/pod-security-admission/api"
+)
+
+var _ = ginkgo.Describe("[rwx-nohci-singlevc-disruptive] RWX-Topology-NoHciMesh-SingleVc-Disruptive", func() {
+	f := framework.NewDefaultFramework("rwx-nohci-singlevc-disruptive")
+	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
+	var (
+		client                  clientset.Interface
+		namespace               string
+		bindingModeImm          storagev1.VolumeBindingMode
+		topologyAffinityDetails map[string][]string
+		topologyCategories      []string
+		leafNode                int
+		leafNodeTag0            int
+		topologyLength          int
+		scParameters            map[string]string
+		accessmode              v1.PersistentVolumeAccessMode
+		labelsMap               map[string]string
+		replica                 int32
+		createPvcItr            int
+		createDepItr            int
+		pvclaim                 *v1.PersistentVolumeClaim
+		allowedTopologies       []v1.TopologySelectorLabelRequirement
+		err                     error
+		topologySetupType       string
+		nodeSelectorTerms       map[string]string
+		allowedTopologyForPod   []v1.TopologySelectorLabelRequirement
+		depl                    []*appsv1.Deployment
+		topologyClusterList     []string
+		powerOffHostsList       []string
+		nimbusGeneratedK8sVmPwd string
+		sshClientConfig         *ssh.ClientConfig
+		stsReplicas             int
+		nodeList                *v1.NodeList
+		noPodsToDeploy          int
+		clusterComputeResource  []*object.ClusterComputeResource
+		noOfHostToBringDown     int
+		podList                 []*v1.Pod
+		service                 *v1.Service
+		statefulset             *appsv1.StatefulSet
+	)
+
+	ginkgo.BeforeEach(func() {
+		client = f.ClientSet
+		namespace = f.Namespace.Name
+
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		// connecting to vc
+		bootstrap()
+
+		// fetch list of k8s nodes
+		nodeList, err = fnodes.GetReadySchedulableNodes(ctx, f.ClientSet)
+		framework.ExpectNoError(err, "Unable to find ready and schedulable Node")
+		if !(len(nodeList.Items) > 0) {
+			framework.Failf("Unable to find ready and schedulable Node")
+		}
+
+		sc, err := client.StorageV1().StorageClasses().Get(ctx, defaultNginxStorageClassName, metav1.GetOptions{})
+		if err == nil && sc != nil {
+			gomega.Expect(client.StorageV1().StorageClasses().Delete(ctx, sc.Name,
+				*metav1.NewDeleteOptions(0))).NotTo(gomega.HaveOccurred())
+		}
+
+		//global variables
+		scParameters = make(map[string]string)
+		labelsMap = make(map[string]string)
+		accessmode = v1.ReadWriteMany
+		bindingModeImm = storagev1.VolumeBindingImmediate
+		topologyClusterNames := GetAndExpectStringEnvVar(topologyCluster)
+		topologyClusterList = ListTopologyClusterNames(topologyClusterNames)
+		nimbusGeneratedK8sVmPwd = GetAndExpectStringEnvVar(nimbusK8sVmPwd)
+
+		// reading testbedingo json
+		readVcEsxIpsViaTestbedInfoJson(GetAndExpectStringEnvVar(envTestbedInfoJsonPath))
```

Comment:
where's value getting assigned or used?

Reply:
This is a util, it will read a testbed info json file and store all the values in a struct, which later I am reading this struct for performing operations on esxi hosts